### ISSUE TITLE
Glass shader normals broken if using color instead of texture

### ISSUE THREAD

**Issue by Gmod4phun** [Facepunch/sbox-issues#5824]
### Describe the bug

Materials using the glass shader are broken when the normal is a color instead of a texture.

### To Reproduce

1. Have a glass shader material
2. Switch between the color and texture option for the default normal
3. Notice the color one being broken

### Expected behavior

Should be working as intended.

### Media/Files

Default normal color:
![image](https://github.com/user-attachments/assets/01931359-0309-4284-8e70-f4e9b62cdb13)

Default normal texture:
![image](https://github.com/user-attachments/assets/87226000-a8c0-4f6c-846c-451046899a26)

**Comment by samzanemesis** [comment]
Do you have any more steps to consistently reproduce this? I've had this issue with normals being generated with wrong color in other shader with but I never managed to pin it down

**Comment by Gmod4phun** [comment]
Not at all, this is the only time I noticed the issue. Checked the console for anything suspicious, this is the only thing that looks like it could be something:

![image](https://github.com/user-attachments/assets/b57a13ca-f6ff-4114-b6be-38ba95253934)
